### PR TITLE
Added crashlytics

### LINKED ISSUE #99: Introduce crashlytics

### PR DESCRIPTION
**Pull request by idelcano**
Closes #99

### CODE REVIEW

File: app/src/main/java/org/hisp/dhis/android/eventcapture/MainActivity.java [hunk 1]
```diff
@@ -57,15 +60,14 @@
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
-
+        Fabric.with(this, new Crashlytics());
```
Comment (xurxodev) [inline app/src/main/java/org/hisp/dhis/android/eventcapture/MainActivity.java]: Can you move Crashlytics initialization to custom application class?

**Review by xurxodev — CHANGES_REQUESTED** [review]
@idelcano please review this crashlytics implementation because I'm generate a crash in app but this crash is not visible in fabric panel.

**Comment by idelcano** [comment]
done!

**Review by xurxodev — APPROVED** [review]
thanks @idelcano

**Pull request merged**